gdk/common/utils.py

```diff
@@ -161,16 +161,17 @@ def parse_json_error(err):
     logging.error(f"{err.args[0]}")
     logging.info(f"The error occurs around line {err.lineno}: " + lines[err.lineno - 1].lstrip())
 
-    logging.info("This might be caused by one of the following reasons: ")
     msg = err.msg
 
     for err_msg, causes in syntax_error_message.JSON_LIBRARY_ERROR_MESSAGES.items():
         if err_msg in msg:
+            logging.info("This might be caused by one of the following reasons: ")
             for cause in causes:
                 logging.info("\t " + cause)
+            logging.info("If none of the above is the cause,")
```

Comment:
Nit: IMO, this line can be removed completely as the log below works well in both the cases.